robottelo/vm.py

```diff
@@ -89,6 +89,33 @@ def __init__(
         if tag:
             self._target_image = tag + self._target_image
 
```

Comment:
Maybe

```python
self._target_image = (tag + self._target_image) if tag else None
```

So we ensure `_target_image` attribute always exists with a default value to None?